en/mkdocs.yml

```diff
@@ -442,6 +442,7 @@ nav:
                 - Production Deployment Guidelines: install-and-setup/setup/deployment-best-practices/production-deployment-guidelines.md
                 - Security Guidelines for Production Deployment: install-and-setup/setup/deployment-best-practices/security-guidelines-for-production-deployment.md
                 - Tuning Performance: install-and-setup/setup/deployment-best-practices/tuning-performance.md
+                - Performance and Capacity Planning: install-and-setup/setup/deployment-best-practices/performance-and-capacity-planning.md
```

Comment:
In 3.1.0 and 3.0.0 we have the following breakdown.

          - Performance Tuning and Test Results:  
            - Tuning Performance: install-and-setup/setup/deployment-best-practices/tuning-performance.md
            - Performance Test Results: install-and-setup/setup/deployment-best-practices/performance-test-results.md

Do we need to rename this page to performance-test-results.md and change the heading of the page?
Or do we need to add a redirection as the file name has changed from 3.1.0?